pytissueoptics/rayscattering/opencl/CLObjects.py

```diff
@@ -52,10 +64,19 @@ def dtype(self) -> ...:
     def hostBuffer(self):
         return self._HOST_buffer
 
+    @hostBuffer.setter
+    def hostBuffer(self, value):
+        if isinstance(value, np.ndarray):
+            self._HOST_buffer = value
+
     @property
     def deviceBuffer(self):
         return self._DEVICE_buffer
 
+    @property
+    def size(self) -> int:
+        return len(self._HOST_buffer)
+
 
```

Comment:
I'm not sure this follows our definition. The size in bytes is useful, but that would be `host_buffer.nbytes` . Maybe rename to length (and create another size or nbytes property for debugging). 

I understand length has more meaning in OpenCL than shape since arrays are usually flattened.

Reply:
yes rename to length